app/helpers/mentor_dashboard_helper.rb

```diff
@@ -0,0 +1,10 @@
+module MentorDashboardHelper
+  def time_ago_in_days(time)
+    time = time.to_time
+    if (Time.now.to_datetime - time.to_datetime) * 86400 < 1.day
+      time_ago_in_words(time)
+    else
+      "#{((Time.now - time) / 86400).to_i} days"
```

Comment:
Are you familiar with the concept of "magic numbers" and why they're undesirable?

Here the number 86400 occurs twice, and there's no context as to what it means. The most common way to avoid magic numbers is to define them as well-named constants, and if possible defined with an expression that is obviously not wrong.

For example 86400 is the number of seconds in a day. You could say:

```
SECONDS_PER_DAY = 86400
```

But that still means that the reader has to double-check... is that really the number of seconds in a day?

Better would be:

```
SECONDS_PER_DAY = 60 * 60 * 24
```

The numbers 60 and 24 immediately speak to the reader, which means the reader can glance at it and move on without having to do a double-take.